code-challenges.test.js

```diff
@@ -13,8 +13,46 @@
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
+//pseudo code
+
+// declare a constant variable  and an array parameter for the shuffle action
+// first, declare a variable that removes the first item by using .shift method
+// use conditionals and a for loop for the new array with first item removed
+// declare a random array and use Math.floor and .random to generate random numbers in array
+// declare new array and let it equal array index
+// set array index to equal an array with new random array indexed
+// set the new random array index to equal the newly declared array
+// return the array
+
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+describe('arrShuffle', () => {
+    const expected = ["yellow", "blue", "pink", "green"];
+    test('matches even if received contains additional elements', () => {
+      expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.arrayContaining(expected));
+    });
+    test('does not match if received does not contain expected elements', () => {
+      expect(["purple", "blue", "green", "yellow"]).not.toEqual(expect.arrayContaining(expected));
+    });
+  });
+
+  describe('arrShuffle', () => {
+    const expected = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"];
+    test('matches even if received contains additional elements', () => {
+      expect(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]).toEqual(expect.arrayContaining(expected));
+    });
+    test('does not match if received does not contain expected elements', () => {
+      expect(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine"]).not.toEqual(expect.arrayContaining(expected));
+    });
+  });
+
+//test passed using the expect.arrayContaining(array)
+//will continue to create the funtion and run the test again
```

Comment:
Remember the goal is to have the test fail as it calls on a function that will do the logic for you.